packages/react-await/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { Await, useAwait, awaitClient } from './Await'
+export { awaitOptions } from './awaitOptions'
 
-export type { AwaitProps } from './Await'
+export type { AwaitProps, AwaitOptions } from './Await'
```

Comment:
With the development of 'awaitOptions', I thought that the AwaitOptions type should also be given access, so I allowed it as follows.